.drone.yml

```diff
@@ -16,6 +16,10 @@ steps:
       path: /var/run/docker.sock
 - name: docker-publish
   image: plugins/docker
+  build_args_from_env:
+    - DRONE_TAG
```

Comment:
Please changes this to be a `build_args` entry:
```yaml
- VERSION=${DRONE_TAG}
```

Reply:
I don't think build_args will translate that properly because build_args is a go string as far as i know not a bash script